hooks/pre-commit.d/01_tests

```diff
@@ -2,4 +2,4 @@
 
 echo "Running django tests"
 
-(cd ./dakara_server && python ./manage.py test)
+(cd ./dakara_server && python ./manage.py test --parallel)
```

Comment:
we could also add the parallel option to the travis test run command

Reply:
This has to be verified, but in parallel, I have experienced complete crash of the tests if there is one error. This was with Django 1.9 (which introduced the parallel feature), thougt.